### PR TITLE
clarify snowflake qs steps

### PR DESCRIPTION
**Pull request by mirnawong1**
[per user feedback](https://dbt-labs.slack.com/archives/C019QLR6EJE/p1704816729148369), this pr sets to clarify the followign:

- role in dev credentials must be specified, otherwise you won't be able to query data in ide
- clarifies that you have to create a new branch after initializing project.

this pr also prompted me to raise the following questions for clarification:

- is role really optional? based on my testing, it seems like it's required in the dev credential settings.
- it seems `transformer` isn't accepted (at least for me) and it shoudl be `transform`.

### CODE REVIEW

File: website/docs/guides/snowflake-qs.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ Using Partner Connect allows you to create a complete dbt account with your [Sno
 
         <Snippet path="snowflake-acct-name" />
     
-    * **Role** &mdash; Leave blank for now. You can update this to a default Snowflake role later.
+    * **Role** &mdash; `transform` or `ACCOUNTADMIN`. This tells dbt what Snowflake role should be assumed after connecting to Snowflake.
```
Comment (mirnawong1) [inline website/docs/guides/snowflake-qs.md]:
```suggestion
    * **Role** &mdash; `transform`, `transformer`, or `ACCOUNTADMIN`. This tells dbt what Snowflake role should be assumed after connecting to Snowflake.
```

File: website/docs/guides/snowflake-qs.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ Using Partner Connect allows you to create a complete dbt account with your [Sno
 
         <Snippet path="snowflake-acct-name" />
     
-    * **Role** &mdash; Leave blank for now. You can update this to a default Snowflake role later.
+    * **Role** &mdash; `transform`, `transformer`, or `ACCOUNTADMIN`. This tells dbt what Snowflake role should be assumed after connecting to Snowflake.
```
Comment (mirnawong1) [inline website/docs/guides/snowflake-qs.md]:
```suggestion
    * **Role** &mdash; `ACCOUNTADMIN`. This tells dbt what Snowflake role should be assumed after connecting to Snowflake.
```

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 this looks great!

I have no suggestions for this. I've approved this for you.

Kind Regards
Natalie

File: website/docs/guides/snowflake-qs.md [hunk 3]
```diff
@@ -191,7 +191,7 @@ Using Partner Connect allows you to create a complete dbt account with your [Sno
 
         <Snippet path="snowflake-acct-name" />
     
-    * **Role** &mdash; Leave blank for now. You can update this to a default Snowflake role later.
+    * **Role** &mdash; `ACCOUNTADMIN`. This tells dbt what Snowflake role should be assumed after connecting to Snowflake.
```
Comment (mirnawong1) [inline website/docs/guides/snowflake-qs.md]:
```suggestion
    * **Role** &mdash; `TRANSFORMER` or your default Snowflake role. This tells dbt what Snowflake role should be assumed after connecting to Snowflake.
```

File: website/docs/guides/snowflake-qs.md [hunk 4]
```diff
@@ -222,8 +222,9 @@ Now that you have a repository configured, you can initialize your project and s
 
 1. Click **Start developing in the IDE**. It might take a few minutes for your project to spin up for the first time as it establishes your git connection, clones your repo, and tests the connection to the warehouse.
 2. Above the file tree to the left, click **Initialize your project**. This builds out your folder structure with example models.
-3. Make your initial commit by clicking **Commit and sync**. Use the commit message `initial commit`. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
-4. You can now directly query data from your warehouse and execute `dbt run`. You can try this out now:
+3. Make your initial commit by clicking **Commit and sync**. Use the commit message `initial commit`. This creates the first commit to your managed repo and allows you to create a new branch where you can add new dbt code. Note, the main branch is set to read-only mode.
```
Comment (nataliefiann) [inline website/docs/guides/snowflake-qs.md]: Hiya @mirnawong1 - I've updated this section following your suggestion
```suggestion
3. Make your initial commit by clicking Commit and sync. Use the commit message initial commit. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
```

File: website/docs/guides/snowflake-qs.md [hunk 5]
```diff
@@ -222,8 +222,9 @@ Now that you have a repository configured, you can initialize your project and s
 
 1. Click **Start developing in the IDE**. It might take a few minutes for your project to spin up for the first time as it establishes your git connection, clones your repo, and tests the connection to the warehouse.
 2. Above the file tree to the left, click **Initialize your project**. This builds out your folder structure with example models.
-3. Make your initial commit by clicking **Commit and sync**. Use the commit message `initial commit`. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
-4. You can now directly query data from your warehouse and execute `dbt run`. You can try this out now:
+3. Make your initial commit by clicking Commit and sync. Use the commit message initial commit. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
+4. Create a new branch by clicking the **+ Create new branch** button below the **Version control** section.
```
Comment (nataliefiann) [inline website/docs/guides/snowflake-qs.md]: @mirnawong1 - I've also updated this section too following your guidance
```suggestion
4. You can now directly query data from your warehouse and execute dbt run. You can try this out now:

- Click + Create new file, add this query to the new file, and click Save as to save the new file:

select * from raw.jaffle_shop.customers

- In the command line bar at the bottom, enter dbt run and click Enter. You should see a dbt run succeeded message.
```

**Comment by matthewshaver** [comment]
@mirnawong1 is this ready to merge?

**Comment by mirnawong1** [comment]
will give it a looksie next week!

**Comment by mirnawong1** [comment]
closing PR as it's stale. if necessary, we can reopen

**Pull request closed without merging**